listenbrainz/webserver/static/js/src/CBReviewModal.tsx

```diff
@@ -0,0 +1,703 @@
+          <b>{listen.track_metadata?.artist_name}.</b> :(
+        </div>
+      );
+    }
+
+    /* The default modal body */
+    const allEntities = [recordingEntity, artistEntity, releaseGroupEntity];
+
+    return (
+      <div>
+        {/* Show warning when text input is too short */}
+        {reviewValidateAlert && (
+          <div className="alert alert-danger">{reviewValidateAlert}</div>
+        )}
+
+        {/* Show warning when recordingEntity is not availible */}
+        {!recordingEntity && (
+          <div className="alert alert-danger">
+            We could not find the reviewable recording entity for{" "}
+            <b>{listen.track_metadata?.track_name}</b>.
+          </div>
+        )}
+
+        <div id="dropdown-container">
+          You are reviewing
+          <span className="dropdown">
+            <button
+              className="dropdown-toggle btn-transparent"
+              data-toggle="dropdown"
+              type="button"
+            >
+              {`${entityToReview.name} 
+              (${entityToReview.type.replace("_", " ")})`}
+              <span className="caret" />
+            </button>
+
+            <ul className="dropdown-menu" role="menu">
+              {/* Map entity to dropdown option button */}
+              {allEntities.map((entity) => {
+                if (entity) {
+                  return (
+                    <button
+                      key={entity.mbid}
+                      onClick={() => this.setState({ entityToReview: entity })}
+                      type="button"
+                    >
+                      {`${entity.name} (${entity.type.replace("_", " ")})`}
+                    </button>
+                  );
+                }
+                return <></>;
+              })}
+            </ul>
+          </span>
+          for <a href={this.CBBaseUrl}>CritiqueBrainz</a>.{this.CBInfoButton}
+        </div>
+
+        <div className="form-group">
+          <textarea
+            className="form-control"
+            id="review-text"
+            placeholder={`Review length must be at least ${this.minTextLength} characters.`}
+            value={textContent}
+            name="review-text"
+            onChange={this.handleTextInputChange}
+            rows={6}
+            style={{ resize: "vertical" }}
+            spellCheck="false"
+            required
+          />
+        </div>
+        <small style={{ display: "block", textAlign: "right" }}>
+          Words: {countWords(textContent)} / Chars: {textContent.length}
+        </small>
+
+        <div className="rating-container">
+          <b>Rating: </b>
+          <Rating
+            className="rating-stars"
+            onClick={(rate: number) => this.setState({ rating: rate })}
+            ratingValue={rating}
+            transition
+            size={20}
+            stars={5}
+          />
+        </div>
+
+        <div className="dropdown">
+          <b>Language: </b>
```

Comment:
Do you think this could be misinterpreted?
Is a clarification like this required?
```suggestion
          <b>Language of your review: </b>
```